Description: Disable broken unit test that fails if two subsequent
 random numbers are the same.
Author: Felix Geyer <debfx-pkg@fobos.de>

diff -Nur qca2-2.0.3.orig/unittest/checkall qca2-2.0.3/unittest/checkall
--- qca2-2.0.3.orig/unittest/checkall
+++ qca2-2.0.3/unittest/checkall
@@ -20,7 +20,7 @@
 cd pgpunittest && make test && cd .. && \
 cd pipeunittest && make test && cd .. && \
 cd pkits && make test && cd .. && \
-cd randomunittest && make test && cd .. && \
+#cd randomunittest && make test && cd .. && \
 cd rsaunittest && make test && cd .. && \
 cd securearrayunittest && make test && cd .. && \
 cd staticunittest && make test && cd .. && \
